demos/whisper-base/main.js

```diff
@@ -105,6 +105,9 @@ let subAudioChunkLength = 0; // length of a sub audio chunk
 let subText = "";
 let speechToText = "";
 
+let time_to_first_token = 0; // TTFT
```

Comment:
```suggestion
let timeToFirstToken = 0; // TTFT
```

these_two_use_snake_case🐍 when theOthersAboveAndBelowUseCamelCase🐪. (yes, whisper.js is aberrant and can do its own thing, but it's good to be consistent within a given file main.js)

```
let maxUnprocessedAudioLength = 0;
let maxProcessAudioBufferLength = 0;
let accumulateSubChunks = false;
let silenceAudioCounter = 0;
...
const blacklistTags = [
```